### DISCUSSION TITLE
Assigning a second set of blocks to the editor

### DISCUSSION THREAD

**Discussion by craigfairhurst8** [GrapesJS/grapesjs#5625] [Ideas]
Hi

I wish to add a second group of blocks to the editor to be rendered to another element. What am I missing / what can I do here?

Thank you

**Comment by artf** [comment]
[Here](https://grapesjs.com/docs/modules/Blocks.html#customization) you can see how to create your own Block Manager

**Comment by craigfairhurst8** [comment]
Brilliant, thanks,
